src/renderer/components/Network/NetworkModal.vue

```diff
@@ -651,6 +667,14 @@ export default {
           return !this.showFull || /^https?:\/\//.test(value)
         }
       },
+      knownWalletsUrl: {
+        required () {
+          return true
```

Comment:
I don't think this field should be required, otherwise the wallet won't be able to add bridgechains without it 🤔

Reply:
It always validates to true, it's basically a "Hack" so i could use the required url field error method.

Reply:
so it does :trollface: